packages/embark/src/lib/modules/blockchain_process/parityClient.js

```diff
@@ -221,13 +235,16 @@ class ParityClient {
       cmd.push("--ws-port=" + config.wsPort);
       cmd.push("--ws-interface=" + (config.wsHost === 'localhost' ? 'local' : config.wsHost));
       if (config.wsOrigins) {
-        if (config.wsOrigins === '*') {
+        const origins = config.wsOrigins.split(',');
+        if (origins.includes('*')) {
           console.warn('==================================');
           console.warn(__('wsOrigins set to "all"'));
           console.warn(__('make sure you know what you are doing'));
           console.warn('==================================');
+          cmd.push("--ws-origins=*");
```

Comment:
According to Parity's docs, `*` should be `all` here: https://wiki.parity.io/Configuring-Parity-Ethereum.html#cli-options

```
    --ws-origins=[URL]
        Specify Origin header values allowed to connect. Special
        options: "all", "none". (default: parity://*,chrome
        -extension://*,moz-extension://*)
```

Which is also what we've done prior your changes in line 230